committeeoversightapp/templates/committeeoversightapp/committee_detail_page.html

```diff
@@ -1,32 +1,167 @@
 {% extends "base.html" %}
+{% load static %}
+
+{% block extra_js %}
+  <script src="https://code.highcharts.com/highcharts.js"></script>
+  <script src="{% static 'js/chart_helper.js' %}"></script>
+{% endblock %}
 
 {% block content %}
-  <div class="col-12 text-center">
-    <h2 class="mb-4">{{ page.title }}</h2>
-    <p class="lead font-weight-light text-dark"><strong>Chairperson:</strong> John Smith</p>
+  <div class="row justify-content-center">
+    <div class="col-12 col-md-10 text-center">
+      <h2 class="my-2">{{ page.title }}</h2>
+      <p class="lead font-weight-light mb-2 mt-3"><strong>Chair: </strong> {{ page.chair }}</p>
+    </div>
   </div>
 
-  <div class="col-md-6 my-3 font-weight-light">
-    {{ page.body|safe }}
+  <div class="row justify-content-center align-items-center">
+    <div class="col-6 col-md-4 col-lg-3 col-xl-3 my-3 pr-4 text-right border-right">
+      <h3 class="rating-large b-plus-rating">B+</h3>
+      <p>
+        Projected CHP Score for the 116th Congress<br />
```

Comment:
Should 116th update automatically?

Reply:
Yeah! I opened a new issue to track that once we get into setting up the actual ratings calculation: #109